aten/src/ATen/native/Math.h

```diff
@@ -2113,4 +2113,15 @@ calc_erfcx(T x)
   }
 }
 
+// TODO: add header here
+template <typename T>
+static inline C10_HOST_DEVICE T calc_log_ndtr(T x) {
```

Comment:
scipy's implementation is only supporting double type, for the sake of higher precision. 
Shall I promote this to acc_type first ?

Reply:
PyTorch tries not to secretly use a different computation type than the input type. There are a few cases where we do promote or have distinct accumulation types, but it's rare. 

For the special operations I don't think we'd want to automatically promote to double unless the float variant of the function was consistently inaccurate, and if we did so we'd want to be sure to make the documentation very clear that the computation was always in double.